### DISCUSSION TITLE
Table Row Number

### DISCUSSION THREAD

**Discussion by sansan061100** [filamentphp/filament#5758] [General]
how to add serial number in each table row

![Screenshot_30](https://user-images.githubusercontent.com/44963978/220563016-87d12203-9f51-4693-8e5c-ee0caaad3040.png)

**Comment by danharrin** [comment]
Search "row index" in the docs

**Reply by sansan061100** [reply]
ok thanks

**Comment by AbdulTawab** [comment]
Try this in $table

use Filament\Tables\Columns\TextColumn;
 
TextColumn::make('index')
    ->label('No. ')
    ->rowIndex()
